db/schema.rb

```diff
@@ -88,6 +88,19 @@
   add_index "bundle_products", ["bundle_product_id"], name: "fk_bundle_prod_prod", using: :btree
   add_index "bundle_products", ["product_id"], name: "fk_bundle_prod_bundle", using: :btree
 
+  create_table "card_readers", force: :cascade do |t|
```

Comment:
On second thought (aka shower thought): do we need two models? We can probably get away with a single model with a controller id and a reader id (and those might have been what you were thinking as opposed to database ids, which is how I read them). I suspect that would make it much easier to build the crud interface and then even add to a product. Having to enter the controller id twice in the interface isn't a big deal in my mind.